documentation/wiki/pages/Polymer Editor - Open file.md

```diff
@@ -26,22 +27,24 @@ Glossary: [Peptide Glossary](https://kb.epam.com/display/EPMLSOP/Peptide+Glossar
-|       |                                 |                                                | **empty file or file without HELM/mol notation selected  <br>AND the opening was confirmed** | **error message is shown "Given string could not be loaded as  molecule. Please validate contents of the file"** |
-|       |                                 |                                                | **option 'paste the text from the clipboard ' is selected<br/>AND <br/> information pasted not in helm/mol  format** <br/>**AND the opening was confirmed** | **error message is shown**  **"Given string could not be loaded as  molecule. Please make sure the string in the  HELM/mol notation"** |
-|       |                                 |                                                | **selecting the supported format, which includes names of monomers not from Ketcher's library<br/>AND the opening was confirmed** | **Ketcher creates the structure for the canvas  for known monomers <br>AND messege shown "Part of monomers was not recognized and was not added"** |
-
+| №    | **User Group**                  | **GIVEN**                                      | **WHEN**                                                     | **THEN**                                                     |
+| ---- | ------------------------------- | ---------------------------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| 1    | Ketcher **polymer** editor User | **Polymer** editor mode of the Ketcher enabled | viewing on the tool panel                                    | I see the Open icon with tooltip "Open... (Ctrl + O) " when hovering on it |
+| 2    |                                 |                                                | clicking on the Open icon <br>OR <br>using the shortcut Ctrl + O | Ketcher provides the ability to select file from the workstation <br> **OR drag file**<br> OR paste the text from the clipboard  <br>AND <br> Ketcher Polymer Editor allows to select  *.mol, *.helm files by default with the ability to select all files<br> |
+| 3    |                                 |                                                | clicking on the Open icon <br/>OR <br/>using the shortcut Ctrl + O<br/>AND<br/>way of adding file is selected | Ketcher provides the ability to add structure to Canvas<br/>OR<br/>Open structure as a new project |
+| 3.1  |                                 |                                                | conditions from the no3  are fulfilled<br/>AND<br/>'Add to Canvas' option was selected | Ketcher Polymer Editor checks that structure opened is a polymer one<br>AND <br/>Canvas in the same state as it was before adding new structure <br/>AND<br/>Ketcher creates the new structure which is moved on canvas by the user's cursor. The first cursor click determines the location of the structure on the canvas |
```

Comment:
We should first check that we can open the structure and then suggest to select the opening mode